### ISSUE TITLE
WCAG 1.3.1: Ensures <li> elements are used semantically (li[aria-hidden="false"])

### ISSUE THREAD

**Issue by oklimanova** [oklimanova/hello-world#19]
#### Issue
`Ensures <li> elements are used semantically` ([listitem](https://dequeuniversity.com/rules/axe/3.5/listitem?application=msftAI))
#### Target application
[Public Comment](https://training.reginfo.gov/public/do/PRA/icrPublicCommentRequest?ref_nbr=202003-1212-001)
#### Element path
li[aria-hidden="false"]
#### Snippet
`<li aria-hidden="false" aria-disabled="false"><a href="#next" role="menuitem" class="btn disabled btn-secondary" id="nextButton">Next</a></li>`
#### How to fix
    Fix any of the following:
      List item does not have a <ul>, <ol> parent element without a role, or a role="list"
#### Environment
Chrome version 84.0.4147.30
This accessibility issue was found using Accessibility Insights for Web 2.18.2 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
